.scrutinizer.yml

```diff
@@ -18,7 +13,6 @@ tools:
     php_loc:
         excluded_dirs:
             - vendor
-            - tests
     php_pdepend: true
     php_sim: true
 build_failure_conditions:
```

Comment:
Can we raise the scrutinizer requirements since you are at 9.2 now?

Reply:
i tried all weekend to get it higher

Reply:
but yeah, i'll update the fail conditions